### DISCUSSION TITLE
face_recognition_model.bin

### DISCUSSION THREAD

**Discussion by perfixel** [vladmandic/face-api#100] [Q&A] [answered]
Hello, I've come across a problem where "sometimes" my browser "chrome" fails to load the "face_recognition_model.bin" file giving me the "net::ERR_BLOCKED_BY_CLIENT" error for that file. I noticed that it's a 6 MB+ file.
In my project I only use the face detection method. I don't use anything else but detecting the faces .. is there a way to minimize this file size or the whole package to suit only my needs? like only detecting faces. no expressions or age or etc
![download (8)](https://user-images.githubusercontent.com/33189075/162345489-38af6768-6069-4108-b6e1-ca28cfa6c60e.png)
?

**Comment by vladmandic** [comment, marked as answer]
this is not a discussion item, it has been addressed via issue #101

**Comment by perfixel** [comment]
Yes sorry, posted it here by a mistake but then added it to the issues section. Also thanks for your response there. I deactivated my adblocker and gonna give it a day to see if the problem occures again.
